test/WorkOSTests/Services/SSO/SSOServiceTest.cs

```diff
@@ -321,5 +321,21 @@ public async void TestGetConnection()
                 JsonConvert.SerializeObject(this.mockConnection),
                 JsonConvert.SerializeObject(response));
         }
+
+        [Fact]
+        public async void TestDeleteConnection()
+        {
+            this.httpMock.MockResponse(
+                HttpMethod.Delete,
+                $"/connections/{this.mockConnection.Id}",
+                HttpStatusCode.OK,
+                RequestUtilities.ToJsonString(this.mockConnection));
```

Comment:
Is this test correct?

It seems like the fixture is returning the connection is returning the connection in the response, when really it returns a 204 No Content with an empty body.